src/contrib/cluster/Akka.Cluster.Metrics.Tests.MultiNode/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿// //-----------------------------------------------------------------------
+// // <copyright file="AssemblyInfo.cs" company="Akka.NET Project">
+// //     Copyright (C) 2009-2021 Lightbend Inc. <http://www.lightbend.com>
+// //     Copyright (C) 2013-2021 .NET Foundation <https://github.com/akkadotnet/akka.net>
+// // </copyright>
+// //-----------------------------------------------------------------------
+
+using Xunit;
+
+[assembly: TestFramework("Akka.MultiNode.TestAdapter.MultiNodeTestFramework", "Akka.MultiNode.TestAdapter")]
```

Comment:
This is the one liner that needs to be added to make sure that all classes in the assembly is grouped in a single collection and parallelization is turned off/ignored.

Reply:
Does this need to be added to all projects? I.e. is this part of the installation instructions for the new MNTR?

Reply:
Ah you're right, this needs to be added to the migration/usage guide.

Reply:
No problem - we _might_ be able to automate the inclusion of this through a `.props` file that we can include inside the MNTR packages but for the time being let's add this to the instructions and explain why it's helpful.